sources/platform/storage/usage.md

```diff
@@ -52,7 +52,9 @@ Additionally, you can quickly share the contents and details of your storage by
 
 ![Storage API](./images/overview-api.png)
 
+<!-- vale Microsoft.Dashes = NO -->
 These URLs link to API _endpoints_—the places where your data is stored. Endpoints that allow you to _read_ stored information do not require an [authentication token](/api/v2#/introduction/authentication). Calls are authenticated using a hard-to-guess ID, allowing for secure sharing. However, operations such as _update_ or _delete_ require the authentication token.
```

Comment:
There is a correct m-dash, but `Microsoft.Dashes` apparently isn't able to overcome the fact that the left side is marked up as italics.